src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.product-card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  line-height: 14px;
+  font-weight: 400;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--catalog {
+    margin-right: $card-margin-right;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    color: $color-gray;
+
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    display: flex;
+
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    line-height: 18px;
+
+    &-text {
+      font-size: 12px;
+      color: $color-gray;
+    }
+
+    &-amount {
+      font-weight: 700;
```

Comment:
Don't use `&` to take part of class name, use it for pseudoclasses, pseudoelements, or for specificity increasing, not for class name combining